dbms/src/TiDB/Schema/SchemaNameMapper.h

```diff
@@ -26,18 +26,56 @@ struct SchemaNameMapper
 
     static constexpr auto DATABASE_PREFIX = "db_";
     static constexpr auto TABLE_PREFIX = "t_";
+    static constexpr auto KEYSPACE_PREFIX = "ks_";
```

Comment:
```suggestion
    static constexpr std::string_view KEYSPACE_PREFIX = "ks_";
```

Reply:
I found the `KEYSPACE_PREFIX` is inferred as `char [4]` which has no allocation in the heap.

Reply:
`static constexpr std::string_view` also don't involve heap allocation, it is treated almost the same as `char[4]` at compile time. But we can use `KEYSPACE_PREFIX.length()` that is decide at compile time without calling `strlen` at runtime.